src/fft.jl

```diff
@@ -541,10 +541,10 @@ for (Tr,Tc,fftw,lib) in ((:Float64,:Complex128,"fftw",libfftw),
         return rFFTWPlan{$Tc,$BACKWARD,inplace,N}(plan, flags, R, X, Y)
     end
 
-    @eval function (::Type{r2rFFTWPlan{$Tr,Any,inplace,N}})(X::StridedArray{$Tr,N},
-                                                            Y::StridedArray{$Tr,N},
-                                                            region, kinds, flags::Integer,
-                                                            timelimit::Real) where {inplace,N}
+    @eval function (::Type{r2rFFTWPlan{$Tr,T,inplace,N}} where T)(X::StridedArray{$Tr,N},
```

Comment:
This looks a little odd, why not place this with the type parameters on line 547? Likewise below.